physicsnemo/utils/profiling/line_profile.py

```diff
@@ -89,7 +89,7 @@ def finalize(self, output_top: Path) -> None:
         # Get the output directory:
         out_top = self.output_dir(output_top)
         with open(out_top / Path("profiler_stats.txt"), "w") as stats:
-            self._profiler.print_stats(stream=stats)
+            self._profiler.print_stats(stream=stats, stripzeros=True)
```

Comment:
Would it be desirable to have `stripzeros` as an argument to `finalize`?

Alternatively, unpacking `kwargs` into `print_stats` to access other functionality?

Reply:
No - if it's going to be configurable, it should go into a config_override for this profiling module.  IMO we should set it to `True` by default.  The only information gained with using it as False is to see if your function that you instrumented was not called.  But if it's not called, that information is already clear since the function will not show up in the outputs.